src/browserbehavior/BrowserBehavior.ts

```diff
@@ -96,4 +96,14 @@ export default interface BrowserBehavior {
    * Returns whether browser supports setSinkId operation
    */
   supportsSetSinkId(): boolean;
+
+  /**
+   * Returns whether browser supports the playback of canvas-captured stream.
+   */
+  supportsCanvasCapturedStreamPlayback(): boolean;
+
+  /**
+   * Returns whether browser supports video processing based on tensforflow models.
```

Comment:
s/tensforflow/TensorFlow

But this is very precise and subject-specific, no? Why is this in our stable, public `BrowserBehavior` interface?

Reply:
Yeah, I hesitated as well.
Or we could have it as manual detection in the demo instead of in browser behavior ?

Reply:
I'd push the logic as close to the use of the code as possible. It's entirely possible that you're hitting one specific bug in one specific thing you're doing, and it's not a general property that customers should use — I find it hard to believe that TF has awful memory leaks _only in Firefox_ and nobody has noticed.

Reply:
https://github.com/tensorflow/tfjs/issues/3319 issue is here...

Reply:
https://github.com/tensorflow/tfjs/issues/3319 issue is here

Reply:
Just to be clear: link to that goes in *the code*.